apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ts

```diff
@@ -43,38 +36,46 @@ router.get(
       res.locals.course_instance,
       res.locals.course,
     );
-    res.locals.groupsCsvFilename = prefix + 'groups.csv';
+    const groupsCsvFilename = prefix + 'groups.csv';
 
-    const groupConfig = await sqldb.queryOptionalRow(
+    const groupConfigInfo = await sqldb.queryOptionalRow(
       sql.config_info,
       { assessment_id: res.locals.assessment.id },
-      GroupConfigSchema,
+      GroupConfigInfoSchema,
     );
-    res.locals.isGroup = !!groupConfig;
-    if (!groupConfig) {
-      res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+    res.locals.isGroup = !!groupConfigInfo;
```

Comment:
This could be a separate variable, or just make this same check in the html.ts file.

Reply:
Now making this check in the html file in [39b5e80](https://github.com/PrairieLearn/PrairieLearn/pull/9550/commits/39b5e80a1ce76beee898fed6daf650f981c27704).